sunpy/map/mapbase.py

```diff
@@ -375,6 +377,44 @@ def center(self):
                                    self.reference_coordinate.y))
 
     @property
+    def shift_amount(self):
```

Comment:
This doesn't seem like a great name, but I can't think of something I like either!!